### ISSUE TITLE
Conda package reference

### ISSUE THREAD

**Issue by zbarry** [ericmjl/nxviz#516]
@ericmjl 
I'm guessing that in the docs, you meant to install from the `conda-forge` channel?

(sr) [me@blah users]$ conda install nxviz
Solving environment: failed

PackagesNotFoundError: The following packages are not available from current channels:

  - nxviz

Current channels:

  - https://repo.anaconda.com/pkgs/main/linux-64
  - https://repo.anaconda.com/pkgs/main/noarch
  - https://repo.anaconda.com/pkgs/free/linux-64
  - https://repo.anaconda.com/pkgs/free/noarch
  - https://repo.anaconda.com/pkgs/r/linux-64
  - https://repo.anaconda.com/pkgs/r/noarch
  - https://repo.anaconda.com/pkgs/pro/linux-64
  - https://repo.anaconda.com/pkgs/pro/noarch

To search for alternate channels that may provide the conda package you're
looking for, navigate to

    https://anaconda.org

and use the search bar at the top of the page.

**Comment by ericmjl** [comment]
@zbarry yes, you are right. It’s supposed to be installed from conda-forge.

**Comment by ericmjl** [comment]
Reminds me, I also need to cut patch releases for the dependencies.

**Comment by zbarry** [comment]
Meaning the line in the docs should be corrected to: `conda install -c conda-forge nxviz` then?

**Comment by ericmjl** [comment]
Yes, that is right. 

Internet on the plane is just amazeballers. I might update the docs there.

**Comment by ericmjl** [comment]
🤖 Posted by Eric's AI agent.

Closing as outdated. This conda installation issue is from 2018. The installation instructions have been updated since; nxviz is available on conda-forge.
